docs/classic-ui/templates-global-variables.md

```diff
@@ -0,0 +1,21 @@
+---
+html_meta:
+  "description": "Convenience global Plone variables in templates"
+  "property=og:description": "Convenience global Plone variables in templates"
+  "property=og:title": "Global variables in templates"
+  "keywords": ""
+---
+
+# Global variables in templates
+
+For convenience Plone defines a couple of global variables often used in templates.
+
+- [portal_state](backend-global-utils-portal-state-label)
```

Comment:
```suggestion
- [`portal_state`](backend-global-utils-portal-state-label)
```

Reply:
what is the reason for the extra quoting in headlines and references?